src/views/Search.js

```diff
@@ -1,7 +1,151 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import SearchIcon from '@material-ui/icons/Search';
+import Button from '../components/Button/Button';
+import noImageFound from '../assets/img/no_image_found.png';
+import SingleLineGridList from '../components/Grid/GridList';
+import TextField from '@material-ui/core/TextField';
+import PropTypes from 'prop-types';
+import { search } from '../actions/searchActions';
+import { withStyles } from '@material-ui/core/styles';
 
-const Search = () => {
-  return <h1>Search</h1>;
+const styles = theme => ({
+  container: {
+    display: 'flex',
+    flexWrap: 'wrap',
+    justifyContent: 'flex-end'
+  },
+  textField: {
+    margin: 0,
+    width: 200
+  }
+});
+
+class Search extends Component {
+  state = {
+    searchQuery: ''
+  };
+
+  search = (query, type, limit, offset) => {
+    this.props.search(query, type, limit, offset);
```

Comment:
A config object would be better than separate parameters.

Reply:
it doesn't matter that I have default params? This is how the Search Service is declared
`search = async (query, type = [], limit = 20, offset = 0) => {`